### ISSUE TITLE
Implementation of perceptual loss

### ISSUE THREAD

**Issue by Ganzhi-e** [Taited/sgdiff#5]
Thank you for an outstanding job！

When do you plan to release the training code? In particular, does the perceptual loss in the paper use StableDiffusionPipeline to obtain the generated image after each noise estimation?

Looking forward for your reply.

**Comment by Taited** [comment]
Hi, thank you for raising this issue.

To ensure reproducibility, this project was extensively re-implemented based on MMagic. We anticipate a release date for the training code and dataset in late January or early February.

Please note that SGDiff utilizes **GLIDE** as its backbone, not StableDiffusion. For more information or if you wish to implement the perceptual loss on your own, please refer to [this repository](https://github.com/PITI-Synthesis/PITI).

**Comment by Ganzhi-e** [comment]
Thank you very much！
